Dockerfile

```diff
@@ -2,4 +2,5 @@ FROM yastdevel/ruby
 COPY . /usr/src/app
 # a workaround to allow package building on a non-s390 machine
 RUN sed -i "/^ExclusiveArch:/d" package/*.spec
-
+# add the change so "check:committed" task does not fail
```

Comment:
We want to build the package in Travis, but Travis runs in an x86_86 VM. And because of the `ExclusiveArch` tag in the spec file the build normally fails.

There is hack on the previous line which removes the tag to allow package build. But this change is found by the `rake package` call (by the `check:committed` subtask) as an uncommited change and the build fails.

So this workaround adds the change to index so `rake` does not fail. Not nice but I'm not sure if there is a better solution.